scss/components/forms.scss

```diff
@@ -1391,26 +1391,21 @@
 
     [_type="flStarRating"] .rating-input~.readonly.rating-star .fa {
       &:before {
-        content: "\f005";
         -webkit-text-stroke: 1px map-get($configuration, formReadOnlySelectedFillColor);
       }
 
       // Styles for tablet
       @include above($tabletBreakpoint) {
-        content: "\f005";
+        &:before {
         -webkit-text-stroke: 1px map-get($configuration, formReadOnlySelectedFillColorTablet);
+        }
       }
 
       // Styles for desktop
       @include above($desktopBreakpoint) {
-        content: "\f005";
+        &:before {
```

Comment:
Indentation doesn't meet coding standards. I implemented and merged https://github.com/Fliplet/fliplet-theme-default/pull/198 which adds ESLint and applies the changes from [the formatter](https://gist.github.com/tonytlwu/8decaec74de85f118ed2b8dcf71dfbba) (see `[scss]` settings).

You need to:

1. Merge the changes from `master` to this branch so it's all up to date
2. Check your VSCode settings
3. Verify your changes to make sure they're correct and follow coding standards
4. Test your changes

Reply:
Done Changes accordingly, taken latest, added VS Code settings.